jstests/aggregation/bugs/server33149.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+    "use strict";
+
+    // For arrayEq.
+    load("jstests/aggregation/extras/utils.js");
+
+    const indexDB = db.getSiblingDB("create_indexess_db");
+    assert.commandWorked(indexDB.dropDatabase());
+
+    // Create an index with key: {x : 1}, name : "x_1
```

Comment:
This is a total nitpick, sorry.

For the comments in this file, please be sure to use full sentences, and end with period. Also, avoid making statements like "Duplicate index cannot be created" about what will or should happen, instead phrasing it with "Test that a duplicate index cannot be created".

Also, please wrap at 100 characters, and only use two lines if the thoughts/comments are very distinct. In your case, I think all the comments should be on one line.